File: python/1_Two_Sum.py
<<<<<<< HEAD
class Solution:
    def twoSum(self,nums, target):
        for i in range(1,len(nums)):
            setter = target - nums[i]
            if setter in nums:
                index = nums.index(setter)
                if i != index:
                    return(i,index)
                    break
=======
class Solution(object):
    def isPalindrome(self, x):
        if x < 0:
            return False

        else:
            number_str = str(x)
            if number_str == number_str[::-1]:
                return True
            else:
                return False
>>>>>>> 2721809cf6a99fcdc31ba6e71c9ea30a5bbb4f0b

File: Go/1_Two_Sum.go
func twoSum(nums []int, target int) []int { 
    for i:=0; i< len(nums); i++{
		x := target - nums[i]
		if slices.Contains(nums,x){
                var index int = slices.Index(nums,x)
                if i != index{return []int{i,index}}
		}
	}
 return nil
}

